package.json

```diff
@@ -9,8 +9,8 @@
   },
   "scripts": {
     "lint": "eslint . --max-warnings 0",
-    "check": "yarn run lint",
-    "test": "echo 'no tests, yet'"
+    "check": "yarn lint && yarn test",
+    "test": "tape plugins/**/*.test.js"
```

Comment:
`tape` is supported out of the box so it was the quick and easy answer, but I can configure Jest if need be.

Reply:
I personally don't see a problem with this since there's only one file being tested so far. I would be ok keeping it simple for now.